commands/remote.go

```diff
@@ -27,7 +27,7 @@ remote, live applications created with Nanobox.
 
 	// RemoteAddCmd ...
 	RemoteAddCmd = &cobra.Command{
-		Use:   "add",
+		Use:   "add <team-name>/<app-name> [remote-alias]",
```

Comment:
Since `team-name` is technically optional, can you update this to be `[team-name/]<app-name> [remote-alias]`